packages/PackageBuilder/tests/Parameter/ParameterProviderTest.php

```diff
@@ -35,9 +35,12 @@ public function testIncludingYaml(): void
 
         $parameterProvider = $container->get(ParameterProvider::class);
 
-        $this->assertSame([
+        $this->assertContains([
             'one' => 1,
             'two' => 2,
         ], $parameterProvider->provide());
+
+
```

Comment:
double newline 😛

Reply:
Ups :)
Isn't there fixer for that? /cc @carusogabriel